qipr_approver/approver/workflows/project_crud.py

```diff
@@ -62,6 +62,13 @@ def update_project_from_project_form(project, project_form, editing_user):
     clinical_setting = extract_tags(project_form, 'clinical_setting')
     collaborator = extract_tags(project_form, 'collaborator')
     keyword = extract_tags(project_form, 'keyword')
+    mesh_keyword = extract_tags(project_form, 'mesh_keyword')
+
+    update_tags(model=project,
+                tag_property='mesh_keyword',
+                tags=keyword,
+                tag_model=Descriptor,
```

Comment:
This is breaking the projects saving routine. Descriptor is not imported, but once imported, it still throws: "type object 'Descriptor' has no attribute 'tag_property_name'"